src/ReflectionFile.php

```diff
@@ -28,21 +29,21 @@ class ReflectionFile
      *
      * @var string
      */
-    protected $fileName;
+    protected string $fileName;
 
     /**
      * List of namespaces in the file
      *
      * @var ReflectionFileNamespace[]|array
      */
-    protected $fileNamespaces;
+    protected array $fileNamespaces;
 
     /**
      * Top-level nodes for the file
      *
      * @var Node[]
      */
-    private $topLevelNodes;
+    private Node|array $topLevelNodes;
```

Comment:
This variable can't hold `Node` type, as otherwise return type hint for the `getNodes()` method will be violated (it returns only arrays now)

Reply:
https://github.com/goaop/parser-reflection/pull/122/commits/03feb2da4c23e65e5c8399c40621d1eefafa7a83